backend/src/Squidex/Config/Authentication/OAuth2Handler.cs

```diff
@@ -0,0 +1,67 @@
+﻿// ==========================================================================
+//  Squidex Headless CMS
+// ==========================================================================
+//  Copyright (c) Squidex UG (haftungsbeschränkt)
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+using Newtonsoft.Json.Linq;
+
+namespace Squidex.Config.Authentication
+{
+    public sealed class OAuth2Handler : OAuthEvents
+    {
+        public override Task RedirectToAuthorizationEndpoint(RedirectContext<OAuthOptions> context)
+        {
+            context.Response.Redirect(context.RedirectUri);
+
+            return Task.CompletedTask;
+        }
+
+        public override Task TicketReceived(TicketReceivedContext context)
+        {
+            var test = context.Request.Body;
+            return base.TicketReceived(context);
+        }
+
+        public override Task RemoteFailure(RemoteFailureContext context)
```

Comment:
Why do you override it, when you do not have a custom implementation?

Reply:
Ok. I'll remove useless overrides